include/ksmbdtools.h

```diff
@@ -86,8 +86,8 @@ extern struct smbconf_global global_conf;
 
 #define KSMBD_CONF_FILE_MAX		10000
 
-#define PATH_PWDDB	"/etc/ksmbd/ksmbdpwd.db"
-#define PATH_SMBCONF	"/etc/ksmbd/smb.conf"
+#define PATH_PWDDB	SYSCONFDIR "/ksmbd/ksmbdpwd.db"
```

Comment:
How will it affect users who have already been using ksmbd-tools? If they update ksmbd-tools, doesn't it work?

Reply:
@namjaejeon, it's up to the distributions to update how they run `configure`, so it should be a major item in the changelog for the next release. Please read https://github.com/cifsd-team/ksmbd-tools/pull/243#discussion_r901920592 and https://github.com/cifsd-team/ksmbd-tools/pull/243#discussion_r902119631. People who install ksmbd-tools from GitHub will have to manually set sysconfdir to /etc when running `configure`.

Reply:
Understood! Thanks for answer.